cSpell.json

```diff
@@ -0,0 +1,289 @@
+        "Overcommunicate",
+        "OWASP",
+        "pansexual",
+        "passcode",
+        "paystub",
+        "paystubs",
+        "Pegasys",
+        "permissioned",
+        "PERSEC",
+        "Photoshop",
+        "POAM",
+        "PRDS",
+        "Premortem",
+        "PSLF",
+        "pushback",
+        "QASP",
+        "Qualtrics",
+        "Reengineering",
+        "reqs",
+        "résumé",
+        "résumés",
+        "roadmap",
+        "roadmapping",
+        "roadmaps",
+        "rotaters",
+        "roundtable",
+        "routable",
+        "screenshare",
+        "SDLC",
+        "Servicedesk",
+        "servicemember",
+        "SFMTA",
+        "signup",
+        "skillset",
+        "skillsets",
+        "Skillsoft",
+        "Slackbot",
+        "SLDP",
+        "Slidedeck",
+        "Smartsheet",
+        "Snyk",
+        "sportcoats",
+        "standups",
+        "Statuspage",
+        "strategize",
+        "submittals",
+        "telework",
+        "timeboxed",
+        "timeframe",
+        "timeframes",
+        "timesheet",
+        "timesheets",
+        "Tock",
+        "Tocked",
+        "Tocking",
+        "Tokencode",
+        "Touchpoints",
+        "trackpad",
+        "transformative",
+        "unaccrued",
+        "Unarchiving",
+        "underserved",
+        "Uptrends",
+        "USDS",
+        "USGSA",
+        "USWDS",
+        "Ventra",
+        "videoconference",
+        "Visio",
+        "VLTP",
+        "vuln",
+        "vulns",
+        "walkthrough",
+        "WCAG",
+        "Webex",
+        "Whiteboarding",
+        "wireframe",
+        "wireframes",
+        "womxn",
+        "workshopping",
+        "XKCD"
+    ],
+    "ignoreWords": [],
+    "patterns": [
+        {
+            "name": "Markdown links",
+            "pattern": "\\((.*)\\)",
+            "description": ""
+        },
+        {
+            "name": "Markdown code blocks",
+            "pattern": "/^(\\s*`{3,}).*[\\s\\S]*?^\\1/gmx",
+            "description": "Taken from the cSpell example at https://cspell.org/configuration/patterns/#verbose-regular-expressions"
+        },
+        {
+            "name": "Inline code blocks",
+            "pattern": "\\`([^\\`\\r\\n]+?)\\`",
+            "description": "https://stackoverflow.com/questions/41274241/how-to-capture-inline-markdown-code-but-not-a-markdown-code-fence-with-regex"
+        },
+        {
+            "name": "Link contents",
+            "pattern": "\\<a(.*)\\>",
+            "description": ""
+        },
+        {
+            "name": "Snippet references",
+            "pattern": "-- snippet:(.*)",
+            "description": ""
+        },
+        {
+            "name": "Snippet references 2",
+            "pattern": "\\<\\[sample:(.*)",
+            "description": "another kind of snippet reference"
+        },
+        {
+            "name": "Multi-line code blocks",
+            "pattern": "/^\\s*```[\\s\\S]*?^\\s*```/gm"
+        },
```

Comment:
Yes - but in a lazy way. At the time I added it (during a PR for another repo), I didn't put in the time/effort to combine the regex. So one assumes single line, one assumes multi-line, rather than making it work for both scenarios. I imagine it wouldn't be a huge challenge for someone to combine them in the future. For now, I figured as long as they're called out explicitly it's OK